.gitmodules

```diff
@@ -1,4 +1,4 @@
-[submodule "qsym_backend/qsym"]
-	path = runtime/qsym_backend/qsym
-	url = https://github.com/eurecom-s3/qsym.git
-	branch = symcc
+[submodule "runtime"]
+	path = runtime
+	url = https://github.com/eurecom-s3/symcc-rt.git
+	branch = export_symcc_runtime
```

Comment:
Shouldn't this be the main branch?

Reply:
When the `symcc-rt` first PR is merged yes. I put this temporary branch for now for testing purposes, we should change it once we are done with the merge.